models/pathology_nuclei_segmentation_classification/docs/README.md

```diff
@@ -0,0 +1,124 @@
+# Model Overview
+A pre-trained model for simultaneous segmentation and classification of nuclei within multitissue histology images based on CoNSeP data. The details of the model can be found in [1]
+
+## Workflow
+
+The model is trained to simultaneous segment and classify nuclei. Training is done via a two-stage approach. First initialised the model with pre-trained weights on the [ImageNet dataset](https://ieeexplore.ieee.org/document/5206848), trained only the decoders for the first 50 epochs, and then fine-tuned all layers for another 50 epochs.
+
+- Each user is responsible for checking the content of models/datasets and the applicable licenses and determining if suitable for the intended use.The license for the pre-trained model used in examples is different than MONAI license. Please check the source where these weights are obtained from: https://github.com/vqdang/hover_net#data-format
+
+pretrained_model = "https://drive.google.com/u/1/uc?id=1KntZge40tAHgyXmHYVqZZ5d2p_4Qr2l5&export=download"
```

Comment:
HI @KumoLiu , I suggest to rewrite this line since it should be a sentence rather than a line of python code in readme, thanks!

Reply:
Thanks, @yiheng-wang-nv, I have rewritten it in https://github.com/Project-MONAI/model-zoo/pull/245/commits/ec8b93437c931018287a9b4b8e6a6d0670901b1f.